crates/holochain/CHANGELOG.md

```diff
@@ -7,12 +7,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 
+- Adds new functionality to the conductor admin API which returns disk storage information. The storage used by apps is broken down into blobs which are being used by one or more app.
```

Comment:
Personally I'd write this _either_ in one _or_ another changelog.

Reply:
Agreed, I removed it here